cypress/e2e/8-motions.js

```diff
@@ -178,6 +175,65 @@ describe('User can create motions via UAC', () => {
     });
   });
 
+  it('Claiming Stakes', () => {
+    cy.login();
+    cy.visit(`/colony/${Cypress.config().colony.name}`);
+
+    // Get amount of staked tokens
+    cy.getBySel('tokenActivationButton', { timeout: 12000 }).click();
+
+    // Get amount of staked tokens
+    cy.getBySel('stakedTokens', { timeout: 60000 })
+      .invoke('text')
+      .as('initialStakedTokens');
+
+    cy.getBySel('stakesTab').click();
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.getBySel('claimableMotionsList', { timeout: 20000 })
+      .wait(2000) // Wait is required to ensure hash is included
+      .find(`[data-test="goToMotion"]`)
+      .first()
+      .click();
+
+    // Get the staked value being claimed
+    cy.getBySel('stakedValue', { timeout: 60000 })
+      .invoke('text')
+      .as('stakedValue');
+
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.getBySel('claimStakeButton', { timeout: 20000 }).click().wait(10000);
+
+    // to close the gas station
+    cy.getBySel('actionHeading').click();
+
+    // Check that the active tokens are correct
+    cy.getBySel('tokenActivationButton', { timeout: 12000 }).click();
+
+    // function is required for `this` object to work
+    cy.getBySel('stakedTokens', { timeout: 6000 })
+      .invoke('text')
+      .as('nowStakedTokens')
+      .then(function () {
+        const initialStakedTokens = this.initialStakedTokens
+          .split(' ')[0]
+          .split(',')
+          .join('')
+          .split('.')[0];
```

Comment:
What the hell is going on here....

Also:
```js
const [firstStakeTokensElement] = this.initialStakedTokens.split(' ');
const parsedStakedTokens = firstStakeTokensElement.replaceAll(',', '');
const [initialStakedTokens] = parsedStakedTokens.split('.');
```

Reply:
Trying to make the formatted numbers play nice, it is messy. I will refactor.